pyproject.toml

```diff
@@ -27,6 +27,7 @@ packages = [
 "Baseten" = "https://baseten.co"
 
 [tool.poetry.dependencies]
+aiohttp = "^3.10.10"
```

Comment:
do we need to drop httpx? Maybe not required, just curious about bloat. no need add ehre since we wanna roll this out quickly.